ProjectBank/Client/Shared/NavMenu.razor

```diff
@@ -29,6 +29,11 @@
                 <span class="oi oi-list-rich" aria-hidden="true"></span> Projects
             </NavLink>
         </div>
+        <div class="nav-item px-3">
+            <NavLink class="nav-link" href="createproject">
+                <span class="oi oi-list-rich" aria-hidden="true"></span> Create project
```

Comment:
This might cause problems with the navbar in #198.
Maybe we should leave this out and add the href in #198.

Reply:
Alright, then this will be merged after #198 and the merge conflicts will be resolved in this branch